clients/dpd-client/src/lib.rs

```diff
@@ -44,20 +37,16 @@ impl Client {
     pub async fn ensure_nat_entry(
```

Comment:
I'm not sure it makes a ton of sense to layer custom methods on top of the progenitor methods.. but maybe this is fine.